pkg/terraform/gather/ovirt/ip.go

```diff
@@ -55,9 +55,16 @@ func getReportedDevices(c *ovirtsdk4.Connection, vmID string) (*ovirtsdk4.Report
 	return reportedDeviceSlice, nil
 }
 
-func findVirtualMachineIP(c *ovirtsdk4.Connection, moRefValue string) (string, error) {
+func findVirtualMachineIP(moRefValue string) (string, error) {
 
-	reportedDeviceSlice, err := getReportedDevices(c, moRefValue)
+	client, err := ovirt.NewConnection()
+	if err != nil {
+		return "", fmt.Errorf("failed to initialize connection to ovirt-engine's %s", err)
+	}
+	defer client.Close()
+
+
+	reportedDeviceSlice, err := getReportedDevices(client, moRefValue)
 	if err != nil {
 		return "", err
```

Comment:
This error should possibly be wrapped so we have context.

Reply:
done